packages/custom-url/config/lists/custom_urls.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" ?>
+<list xmlns="http://schemas.sulu.io/list-builder/list">
+    <key>custom_urls</key>
+
+    <properties>
+        <property name="id" visibility="no" translation="sulu_admin.id" sortable="false">
+            <field-name>id</field-name>
+            <entity-name>Sulu\CustomUrl\Domain\Model\CustomUrlInterface</entity-name>
+        </property>
+
+        <property name="title" visibility="always" translation="sulu_admin.title" sortable="false" searchability="yes">
+            <field-name>title</field-name>
+            <entity-name>Sulu\CustomUrl\Domain\Model\CustomUrlInterface</entity-name>
+            <filter type="text" />
+        </property>
+
+         <property name="webspace" visibility="never" sortable="false" searchability="never">
+            <field-name>webspace</field-name>
+            <entity-name>Sulu\CustomUrl\Domain\Model\CustomUrlInterface</entity-name>
+            <filter type="text" />
+        </property>
```

Comment:
We don't want to filter for it in the UI only in the code, maybe there is a way?

Reply:
I'm stupid. Fixed